docs/manage/_advanced.rst

```diff
@@ -270,34 +270,34 @@ Alternatively, you can follow these steps in the GUI.
 
 
 
-Customize Kolibri Settings with the OPTION.INI File
----------------------------------------------------
+Customize Kolibri Settings with the OPTIONS.INI File
+----------------------------------------------------
 
-For some advanced configuration settings you need to use the ``option.ini`` file. Installing Kolibri does not generate this file by default, but you can easily add one yourself. Follow these steps.
+For certain configuration settings you need to use the ``options.ini`` file. Installing Kolibri does not generate this file by default, but you can easily add one yourself. Follow these steps.
 
 #. Open the preferred text editor on your computer (eg. Notepad on Windows).
 #. Write the required *sections* and *keys* (see details for available settings below) in the following format:
    
-    .. code-block:: bash
+    .. code-block:: text
```

Comment:
I think this can be `.. code-block:: ini`

Reply:
Done